ethcore/src/evm/interpreter/mod.rs

```diff
@@ -104,12 +104,13 @@ impl<Cost: CostType> evm::Evm for Interpreter<Cost> {
 			position: 0,
 			code: &code
 		};
+		let infos = &*instructions::INSTRUCTIONS;
 
 		while reader.position < code.len() {
 			let instruction = code[reader.position];
 			reader.position += 1;
 
-			let info = instructions::get_info(instruction);
+			let info = infos[instruction as usize];
```

Comment:
instruction is `u8` - array is initialized for `0x100` elements, but maybe it will be better to be more explicity?